.github/workflows/ci.yaml

```diff
@@ -132,3 +139,52 @@ jobs:
         uses: github/codeql-action/upload-sarif@v3
         with:
           sarif_file: ${{ steps.slither.outputs.sarif }}
+
+  mythril:
```

Comment:
The analysis with mythril is not particularly fast. Are we sure we want to set this for every commit?